eventing/README.md

```diff
@@ -18,143 +18,171 @@ The above concerns are consistent with the
-  namespace-scoped, while ClusterEventTypes are installed by an administrator
-  and available in all namespaces in your cluster.
-- **EventSource** and **ClusterEventSource** describe an external system which
-  may produce one or more EventTypes.
-
-We currently have 3 sources implemented:
-
-- [K8sevents](https://github.com/knative/eventing/tree/master/pkg/sources/k8sevents)
-  collects
-  [Kubernetes Events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#event-v1-core)
-  and presents them as CloudEvents.
-  ```bash
-  kubectl apply --filename https://storage.googleapis.com/knative-releases/eventing/latest/release-source-k8sevents.yaml
-  ```
-- [GitHub](https://github.com/knative/eventing/tree/master/pkg/sources/github)
-  collects pull request notifications and presents them as CloudEvents.
-  ```bash
-  kubectl apply --filename https://storage.googleapis.com/knative-releases/eventing/latest/release-source-github.yaml
-  ```
-- [GCP PubSub](https://github.com/knative/eventing/tree/master/pkg/sources/gcppubsub)
-  collects events published to a GCP PubSub topic and presents them as
-  CloudEvents.
-  ```bash
-  kubectl apply --filename https://storage.googleapis.com/knative-releases/eventing/latest/release-source-gcppubsub.yaml
-  ```
-
-### Flows
-
-Lastly, we have a higher-level abstraction called a Flow which bundles up the
-specification from the Source to the endpoint, optionally allowing you to choose
-the Channel and Bus which the event is routed over. (Otherwise, there is a
-default Bus used to provision a Channel.)
-
-- **Flow** is the top-level user-facing concept in Eventing; it describes the
-  desired path from an external Source of events to a destination that will
-  react to the events. There is only a single type of Flow, which is installed
-  by the core Knative Eventing install.
-
-## Supporting Libraries
-
-In addition to the above, there is a golang library at
-[`pkg/event`](https://github.com/knative/eventing/tree/master/pkg/event) for
-handling CloudEvents and easily creating a container which processes CloudEvents
-delivered over HTTP. You can look at several of the samples for usage examples.
+The eventing infrastructure supports two forms of event delivery at the moment:
+
+1. Direct delivery from a source to a single Service. In this case, the Source
+   is responsible for retrying or queueing events if the destination Service is
+   not available.
+2. Fan-out delivery from a source or Service response to multiple endpoints
+   using Channels and Subscriptions. In this case, the Channel implementation
+   ensures that messages are delivered to the requested destinations and should
+   buffer the events if the destination Service is unavailable.
+
+![Control plane object model](control-plane.png)
+
+The actual message forwarding is implemented by multiple data plane components
+which provide observability, persistence, and translation between different
+messaging protocols.
+
```

Comment:
the top part of this `data-plane.png` is not very helpful, pretty much a copy from the previous picture. I would cut it

Reply:
It is a copy from the previous picture. I wasn't sure whether it was clearer by repeating the focused part of the picture or not. I can cut this if it makes sense. (I also wasn't sure when I did the diagram how close the two pictures would be.)